dashboard/src/actions/charts.ts

```diff
@@ -54,17 +55,16 @@ export type ChartsAction = ActionType<typeof allActions[number]>;
 
 async function httpGet(dispatch: Dispatch, targetURL: string): Promise<any> {
   try {
-    const response = await fetch(targetURL);
-    const json = await response.json();
-    if (!response.ok) {
-      const error = json.data || response.statusText;
+    const response = await axios.get(targetURL);
+    if (response.status < 200 || response.status > 299) {
```

Comment:
bit obvious but might be easier to read with a comment
```suggestion
    # if non-2XX response
    if (response.status < 200 || response.status > 299) {
```